src/tree/subscriptionTreeItem.ts

```diff
@@ -73,21 +81,20 @@ class SubscriptionTreeItem extends AzExtParentTreeItem implements SubscriptionTr
         return false;
     }
 
-    public async loadMoreChildrenImpl(): Promise<AzExtTreeItem[]> {
-        const clusterResources = await getResources(
+    async fetchClustersAndFleets() {
```

Comment:
let's be better and consistent in the function signature 
- can we define the return types here
- add private keyword in front of it to be explicit